scipy-1.0/scipy-optimize.tex

```diff
@@ -21,66 +21,8 @@ \subsubsection*{Linear Optimization}
-The \texttt{brute} function performs a grid search over the entire domain, and the best grid point is ``polished'' by a local minimizer. However, this approach becomes prohibitively expensive as the problem dimensionality increases.
-% As far as I understand how brute works, the run time does not depend on the number of extrema, as it only polishes the best one
-% , or if the objective function has many extrema. 
-To address this, the
-\texttt{basinhopping} \cite{Wales1997} function uses random perturbation of the decision variables in conjunction with local minimization. The acceptance of the perturbed coordinates is dictated by the Metropolis test typically used in Monte Carlo simulations. 
-\texttt{differential\textunderscore evolution} \cite{Wormington1999,Storn1997} is a stochastic solver that improves by gradual evolution of a population of candidate solutions. In each iteration, trial candidates are generated by combination of candidates from the existing population; if the trial candidates represent an improvement then the population is updated. 
-% Is the following a unique feature of differential_evolution? If so, state it (although it seems that neither `brute`or `basinhopping` inherently require the a differentiable problem). If not, why mention it? 
-% \texttt{differential\textunderscore evolution} does not require that the objective function be differentiable.
-%  basinhopping may need differentiable objective for the local minimization.
-Most recently, the SciPy benchmark suite gained a comprehensive set of 196 global optimization problems
-% They're ``comprehensive''
-% of varying difficulties, 
-% that can be used to evaluate the performance of the solvers. 
-%This suite is used to decide
-for tracking the performance of existing solvers over time and for evaluating whether the performance of new solvers merits inclusion in the package. 
-% have good enough performance to be added to the package.
+As \texttt{minimize} only seeks a local minimum, some problems require the use of a global optimization routine to find the minimum of the objective function over the set of all decision variable values, especially if the objective function possesses many local minima that other solvers could get stuck in. The new \texttt{scipy.optimize.differential\textunderscore evolution} \cite{Wormington1999,Storn1997} is a stochastic global optimizer that works by evolving a population of candidate solutions. In each iteration, trial candidates are generated by combination of candidates from the existing population. If the trial candidates represent an improvement, then the population is updated. Most recently, the SciPy benchmark suite gained a comprehensive set of 196 global optimization problems for tracking the performance of existing solvers over time and for evaluating whether the performance of new solvers merits inclusion in the package.
```

Comment:
Is there a reason why `basinhopping` was removed?

Reply:
The "Key technical improvements" section is supposed to "describe key technical improvements made in the last three years" (prior to 1.0 branch). `basinhopping` is much older than that. Was it substantially approved within the window?

Reply:
@andyfaff Look OK?